Products/CMFPlone/PloneTool.py

```diff
@@ -142,6 +142,10 @@ def validateSingleNormalizedEmailAddress(self, address):
             # Address contains two newlines (possible spammer relay attack)
             return False
 
+        # address can be empty if getaddresses has parsing errors (returns [("", "")])
```

Comment:
Can we do `address = address.strip()` first, for good measure?  Actually I would do that a few lines above, after the `isinstance` check.

When we are being called by our own `validateSingleEmailAddress` there is probably nothing to strip away, but someone else may call this method.

Reply:
yes good point ... I'll change that.